master/internal/trial.go

```diff
@@ -153,6 +154,21 @@ func (t *trial) Receive(ctx *actor.Context) error {
 				return fmt.Errorf("persisting trial in prestart: %w", err)
 			}
 		}
+
+		if t.continued {
+			// TODO these should be in a transaction eventually.
```

Comment:
are you planning to do this for this PR?

Reply:
no